airflow/models/dag.py

```diff
@@ -454,6 +456,7 @@ def __init__(
         concurrency: int | None = None,
         max_active_tasks: int = airflow_conf.getint("core", "max_active_tasks_per_dag"),
         max_active_runs: int = airflow_conf.getint("core", "max_active_runs_per_dag"),
+        max_consecutive_failed_dag_runs: int | None = None,
```

Comment:
It’s better to call `getint` here as the default instead. The None type can go away too.